src/global/signal_handler.cc

```diff
@@ -179,25 +179,22 @@ struct SignalHandler : public Thread {
   };
 
   /// all handlers
-  safe_handler *handlers[32];
+  safe_handler *handlers[32] = {nullptr};
```

Comment:
I think the `nullptr` will be assigned only to `handlers[0]` and the remaining will be assigned just `NULL` or `0`. It's better not to put `nullptr` there or specify a range.

I mean:
`safe_handler *handlers[32] = {};`
or
`std::fill_n(handlers, 32, nullptr);`

Reply:
@joscollin please note this is a backport PR.